control_planners_benchmark:
  ros__parameters:
    planner_timeout: 50.0
    interpolation_parameter: 50
    octomap_voxel_size: 0.2
    selected_state_space: "SE2" # "DUBINS","REEDS", "SE2", "SE3" ### PS. Use DUBINS OR REEDS for Ackermann
    selected_planners: [RRT,LQRRRTStar]
    min_turning_radius: 1.5
    state_space_boundries:
      minx: -40.0
      maxx: 40.0
      miny: -40.0
      maxy: 40.0
      minz: -5.0
      maxz: 5.0
      minyaw: -3.14
      maxyaw: 3.14
      minv: -0.5
      maxv: 0.5
    control_boundries:
      minv: -0.5
      maxv: 0.5
      minw: -0.5
      maxw: 0.5
    robot_body_dimens:
      x: 1.2
      y: 0.9
      z: 0.8
    start:
      z: 1.5
    goal:
      z: 1.5
    goal_tolerance: 0.4
    min_euclidean_dist_start_to_goal: 25.0
    batch_size: 1
    epochs: 1
    max_memory: 4096
    results_output_dir: "/home/atas/test/"
    results_file_regex: "control"
    publish_a_sample_bencmark: true
    sample_bencmark_plans_topic: "benchmark_plan"
